src/framework/global/containers.h

```diff
@@ -437,4 +443,11 @@ inline void DeleteAll(const Container& c)
 }
 }
 
+template<typename T>
+inline std::set<T>& operator<<(std::set<T>& s, const T& v)
```

Comment:
This is the first container with such an operator, and developers will want to add it for other containers.
Do we want to see such operators for other containers? I think using the insert/push_back operator better shows that we are trying to use std

Reply:
I like this operator for containers...
We have already added many helpers functions to make it easier to use std. I find std quite ugly (